src/main/java/pl/javastart/task/PhoneContract.java

```diff
@@ -0,0 +1,11 @@
+package pl.javastart.task;
+
+public abstract class PhoneContract {
+    abstract boolean sendMmsIfPossible();
+
+    abstract boolean sendSmsIfPossible();
+
+    abstract int talkIfPossible(int secondsExpected);
+
+    abstract void accountState();
```

Comment:
Lepiej, żeby ta metoda zwracała String z opisem. Umowa (powiedzmy karta sim) nie odpowiada za wyświetlanie czegokolwiek. Za to odpowiada telefon, czyli u Ciebie klasa Phone